src/tasks/runstart.ts

```diff
@@ -308,10 +316,12 @@ export const RunStartQuest: Quest = {
     {
       name: "Mummery",
       completed: () =>
-        get("_mummeryMods").includes(`Experience (Mysticality)`) ||
+        get("_mummeryMods").includes(`Experience (${mainStatStr})`) ||
         !have($item`mumming trunk`) ||
         get("instant_saveMummingTrunk", false),
-      do: () => cliExecute("mummery myst"),
+        do: (): void => {
+          cliExecute(`mummery ${mainStatStr}`);
+        },
```

Comment:
I noticed a lot of these `cliExecute`s and `modifiers` switched from `myst` to `${mainStatStr}` (i.e. Mysticality).

I assume this works fine (i.e. `maximize myst` works identically to `maximize mysticality`, and ditto for `mummery myst` -> `mummery mysticality`), but we should do our due dilligence to check all of them (e.g. maximizers, mummery, fortune buff)

Reply:
Agreed. It works on Folger, but that's small comfort.

Reply:
![image](https://github.com/Pantocyclus/InstantSCCS/assets/98746573/5e41e6a0-ca3c-41b3-9d03-c92e43c852af)
It does not work